scripts/core/exp-bindless-images.yml

```diff
@@ -23,6 +23,78 @@ desc: "Handle of interop semaphore"
 class: $xBindlessImages
 name: "$x_exp_interop_semaphore_handle_t"
 --- #--------------------------------------------------------------------------
+type: enum_exp
```

Comment:
What about still calling this `type: enum` but having an additional flag like `extend: true`?

Reply:
I like this idea 👍  - makes a few thing simpler